README.md

```diff
@@ -3864,13 +3864,86 @@ _You can enable the following settings in Xcode by running [this script](resourc
   * If the type in question is a simple value type (e.g. fewer than 20 lines), it is OK to omit the `// MARK:`s, as it would hurt legibility.
 
 * <a id='subsection-organization'></a>(<a href='#subsection-organization'>link</a>) **Within each top-level section, place content in the following order.** This allows a new reader of your code to more easily find what they are looking for. [![SwiftFormat: organizeDeclarations](https://img.shields.io/badge/SwiftFormat-organizeDeclarations-008489.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#organizeDeclarations)
-  * Nested types and type aliases
-  * Static properties
-  * Class properties
-  * Instance properties
-  * Static methods
-  * Class methods
-  * Instance methods
+  - **Nested types and type aliases**: 
+  - **Static properties**: 
+  - **Static property with body**: 
+  - **Class properties with body**: 
+  - **SwiftUI dynamic properties (@State, @Environment, @Binding, etc)**: SwiftUI Dynamic properties should appear grouped since they share a common semantic meaning and cause a SwiftUI view to re-render its body.
+      
+      <details>
```

Comment:
We should have single `<details>` at the bottom of the bullet list, and put all of the code examples there.

Reply:
> SwiftUI Dynamic properties should appear grouped since they share a common semantic meaning and cause a SwiftUI view to re-render its body

I think this could work as a code comment in the example code, and as rationale in the PR description. I don't think we need to put this in the main above-the-fold description of the rule though, normally we put any rationales or additional content within the `<details>` block.